e2e/wdio/core/tests/tile.e2e-spec.ts

```diff
@@ -0,0 +1,114 @@
+import { TilePo } from '../pages/tile.po';
+import {
+    getText,
+    refreshPage,
+    getElementClass,
+    scrollIntoView,
+    getElementArrayLength,
+    setValue,
+    getValue,
+    getAttributeByName,
+    getElementPlaceholder,
+    mouseHoverElement,
+    isElementDisplayed,
+    isElementClickable,
+    click,
+    sendKeys,
+    waitForPresent,
+    acceptAlert,
+} from '../../driver/wdio';
+
+import { sections, states } from '../fixtures/appData/textarea-contents'
+
+describe('Textarea component test', function () {
+    const tilePage = new TilePo();
+    const {
+        tile,
+        fdTile,
+        launchExample,
+        kpiExample, 
+        feedExample,
+        numericContent,
+        actionExample,
+        closeButton, 
+        moreButton,
+        bagdeExample,
+        badge,
+        tileContent,
+        tileFooter,
+        tileHeader,
+        columnsExample
+    } = tilePage;
+
+    beforeAll(() => {
+        tilePage.open();
+    }, 1);
+
+    beforeEach(() => {
+        refreshPage();
+        waitForPresent(tilePage.title);
+    }, 1);
+
+    it('should check that tile clickable', () => {
+        for (let i = 0; i < tile.length; i++) {
+            expect(isElementClickable(tile, i)).toBe(true, `tile with index ${i} is not clickable`);
+        }
+    });
+
+    it('should check size in default example', () => {
+        expect(getAttributeByName(launchExample + fdTile, 'size', 1)).toBe('s');
+        expect(getAttributeByName(launchExample + fdTile, 'size', 3)).toBe('s');
+    })
+
+    it('should check launch example', () => {
+        checkTileType(launchExample, 'launch');
+    });
+
+    it('should check kpi example', () => {
+        checkTileType(kpiExample, 'kpi');
+    });
+
+    it('should check feed example', () => {
+        checkTileType(feedExample, 'feed');
+    });
+
+    it('should check size of kpi example', () => {
+        expect(getAttributeByName(kpiExample + numericContent, 'size', 2)).toBe('m');
+        expect(getAttributeByName(kpiExample + numericContent, 'size', 3)).toBe('m');
+        expect(getAttributeByName(kpiExample + numericContent, 'size', 4)).toBe('s');
+        expect(getAttributeByName(kpiExample + numericContent, 'size', 5)).toBe('s');
```

Comment:
in the cases where you are using a selector like numericContent for only 1 example group, better just to make a specific seelctor instead. I.e.
kpiNumericContent = this.kpiExample + '.fd-numeric-content';

makes the test easier to read
expect(getAttributeByName(kpiNumericContent, 'size', 5)).toBe('s');

Reply:
same for others